src/main/java/com/odysseusinc/arachne/datanode/dto/atlas/PatientLevelPredictionAnalysis.java

```diff
@@ -0,0 +1,1486 @@
+/*
```

Comment:
What benefits we receive having these DTOs? We are not changing imported files, we are not validating them - we pass them as-is, so why to increase code complexity?